sass/prtrails.scss

```diff
@@ -41,18 +42,13 @@ a {
       &.arrow-left {
         position:absolute;
         left:$headerPadding;
-        top:$statusBar-height;
         text-align:left;
       }
-      &.arrow-right {
+      &.header-home {
         position:absolute;
         right:$headerPadding;
-        top:$statusBar-height;
         text-align:right;
```

Comment:
Worth adding "color: black" here, and for .arrow-left, so that it doesn't colour it as a hyperlink.